cypress/support/step_definitions/index.ts

```diff
@@ -24,11 +25,11 @@ const YAML = {
 
 let env = {}
 let selectors: Record<string, string> = {}
-let urls = new Map()
+const client = useClient()
 Before(() => {
   env = {}
   selectors = {}
-  urls = new Map()
+  client.history = []
```

Comment:
I'll likely add a `client.clear()` in a soon to come PR, if we are to use request counting (or anything else to do with a 'client' that we need to reset/clear) we'll need to clear/reset other things too.

Reply:
This one is done